assets/src/edit-story/utils/useUploadVideoFrame.js

```diff
@@ -8,10 +8,12 @@ import { useCallback } from '@wordpress/element';
 import { useAPI } from '../app/api';
 import { useStory } from '../app/story';
 import { useConfig } from '../app/config';
+import { useUploader } from '../app/uploader';
 import getFirstFrameOfVideo from './getFirstFrameOfVideo';
 
 function useUploadVideoFrame( { videoId, src, id } ) {
-	const { actions: { uploadMedia, saveMedia } } = useAPI();
+	const { actions: { saveMedia } } = useAPI();
```

Comment:
Could we perhaps rename`saveMedia` to `updateMedia` to reflect the action more accurately?
`saveMedia` could also mean saving new media (uploading media), would be good to make a clear difference here.

Reply:
`saveMedia` was added in another PR and changing it here seem unrelated here. We can create another issue for this.

Reply:
It's related to the code readability of this PR as well.
Would it be a big effort to change the name here? Not talking about changing the logic, just the name for clarity, not sure it's worth its own issue.

Reply:
It is strange as I have made similar comment in the past about changing name and told you to not make changes like that are unrelated. I have made the change, but this once. https://github.com/google/web-stories-wp/pull/195/commits/7f937d9e6f2acb753223b63b85ded81cd3265d9c